src/read_utils.jl

```diff
@@ -347,6 +347,15 @@ function collect_separable(bandirsd::Dict{String, Vector{Pair{UnitRange{Int}, Ve
                 lgirsd::Dict{String, Vector{LGIrrep{D}}};
                 latestarts::Dict{String, Int}=Dict("Γ" => D)) where D
 
+    # Check for empty values in bandirsd and return gracefully. Otherwise the mapreduce
+    # line will throw an error.
+    for (_, v) in bandirsd
+        if isempty(v)
+            warn("bandirsd has empty values. May indicate an empty lattice.")
+            return nothing
+        end
+    end
```

Comment:
I think it may be better and more consistent if we simply return empty variants of what we are usually returning. The reason being that if we return `nothing`, then we implicitly also ask callers to always have a `tmp = collect_separable(...); if isnothing(tmp); #unwrap tmp; end` step. But if we just return empty containers, we can just check for emptiness (may not even be necessary if we loop over the returned values later) and we avoid the need to unwrap.

Something like this:
```suggestion
   if any(isempty, values(bandirsd))
      # return empty `collectibles_bands, collectibles_symvecs`
      return Vector{UnitRange{Int}}(), Vector{Dict{String, Vector{Int}}}()
   end
```